modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/screen/navigation/entries/general.jsp

```diff
@@ -40,6 +40,15 @@ LayoutType selLayoutType = selLayout.getLayoutType();
 	onSubmit="event.preventDefault();"
 	wrappedFormContent="<%= false %>"
 >
+	<aui:script>
+		var groupElement = document.getElementById('<portlet:namespace />groupLabel');
+
+		groupElement.setAttribute(
+			'aria-labelledby',
+			'<%= LanguageUtil.get(request, "general") %>'
+		);
+	</aui:script>
```

Comment:
Does the position of this script tag matter? I'm typically used to seeing them at the bottom of a file. 

It has been awhile since I have regularly worked in JSPs though...

Reply:
I think it is only preference and shouldn't effect how it runs.
I decided to add it after the `liferay-frontend:edit-form` to make it clear what is being modified.

Reply:
On the technical level, there are two issues here:
- `<aui:script>` without arguments will create vanilla `<script>` tag that will pollute global namespace. We should either add `sandbox="<%= true %>"`, or wrap code in IIFE.
- we should check after any DOM query if the element was found.

But, these points are likely not important, as the setting of `aria-labelledby` does not look right (see the other comment).

Reply:
Hello @markocikos,
Thank you for these two points.

I can make these changes and leave it as label instead.